packages/thumbprint-react/components/Image/index.jsx

```diff
@@ -24,30 +36,15 @@ const Image = forwardRef((props, outerRef) => {
     const [containerRef, setContainerRef] = useState(null);
 
     // --------------------------------------------------------------------------------------------
-    // Inline styles for container `div` and `Picture` component
+    // Sizes
     // --------------------------------------------------------------------------------------------
 
-    const pictureProps = {
-        src,
-        sources,
-        alt,
-        style: { width: '100%', height: height || '100%', display: 'block' },
-        sizes: containerRef && containerRef.clientWidth ? `${containerRef.clientWidth}px` : '0px',
-    };
-
-    const containerProps = {
-        ...rest,
-        style: {
-            ...style,
-            // Allows the container to behave like an image would. Without this, something like
-            // passing a `border-radius` in `className` or `style` would not work since the
-            // container is a `div`, not an `img`.
-            overflow: 'hidden',
-            // Setting a `min-height` makes the lazy-loading work in cases where the `Image` parent
-            // has a `0px` height.
-            minHeight: '1px',
-        },
-    };
+    // Used by srcSet to determine which image in the list will be requested. Unless the image
+    // is full width this value has to be calculated client-side because we don't know the
```

Comment:
> Unless the image is full width

We don't currently provide a way to opt-out of this calculation, so it's always client-side.

Reply:
Ok, removed.

Reply:
Fix was removed in the reverts.

Reply:
Fixed